blueprints/hr_assistant/domains/date/get_date/train.txt

```diff
@@ -0,0 +1,203 @@
+{julia|name} {birthday|dob}
+tell me {birth date|dob} of {Ermine Pelletier|name}
+What was the exact date when {desiree|name} was {hired|employment_action} at our company?
+{start date|employment_action}, {Bianca Roehrich|name}
+when did {Amy|name} officially {join|employment_action} the company
+find me the official {start date|employment_action} of {Nan|name} at this company
+What was it that {Nilson Ferivandes|name} {began working for|employment_action} this company?
+{Quinn Rarrick|name} {bday month|dob}
+{ivan|name} {start date|employment_action}
+When was {Lenora Tejeda|name} {born|dob}?
+{Ivan Rogers|name} {bday month|dob}
+When {Trina Alagbe|name} was {born|dob}, what was the month
+How long ago did we {fire|employment_action} {nan|name} from the company
+How long has Mr.{lin chan|name} been with the company?
+Can you tell me how long has {Carlos|name} worked in the firm
+whats the date of {joining|employment_action} for ms.{Morway|name}
+When was {Julia|name} {fired|employment_action}?
+{Leigh Smith|name} {date of birth|dob}
+What was the exact date when {Amy|name} {started her job|employment_action} at our company?
+What was {Brandon LeBlanc|name}s date when he started to {work here|employment_action}?
+What was {nan|name}'s {date of hiring|employment_action}?
+What year was {ivan rogers|name} {hired|employment_action}?
+Did we {let go|employment_action} of {Amy|name} in {May|sys_time}
+Can you tell me how long has {mia|name} worked in the firm
+{julia|name} was {fired|employment_action} on which date
+{Myriam Givens|name} {date of birth|dob}
+{Julia|name} {firing date|employment_action}
+What was the date when {ivan rogers|name} was {hired|employment_action}
+Has {Sarah Warfield|name} been working here for {3.2 years|sys_duration}?
+{Sean Quinn|name} {date of birth|dob}
+{mia brown|name} {fire date|employment_action}
+When {Amy|name} was {hired|employment_action} to work at the company, what was the date?
+What was the exact date when {Renee Becker|name} {started working here|employment_action} at our company?
+Which month does {Abdellah Veera|name} have his {bday|dob}
+{Amy|name} {date of fire|employment_action}
+{Shana Maurice|name} {bday month|dob}
+{Jordan Winthrop|name} {recruited|employment_action} when
+How long has {Julia Soto|name} worked here?
+I want the date when {julia soto|name} {starting working|employment_action} here
+in what month was {Amy Foster-Baker|name} {born|dob}
+How long was {Melisa Gerke|name} working for?
+{end her employment|employment_action}, {Jyoti Lajiri|name} which day
+{julia|name} {termination date|employment_action}
+What was the {end date|employment_action} for {Sean Quinn|name}?
+When exactly did we let {nan singh|name} go?
+When {Brannon Miller|name} started {working here|employment_action}, what was the date
```

Comment:
same as the previous comment. I can simply replace started with stopped. The entity annotated would not capture whether it's the hire or fire date.